### DISCUSSION TITLE
Search and filter function does not work

### DISCUSSION THREAD

**Discussion by Fraggle78** [YOURLS/YOURLS#3585] [General]
Hello,
I am using version 1.9.2 and I have problems with search and sorting. By default 15 lines are displayed. But when I specify to display 50 lines, I see only 6 lines. No matter how many I want to display, it remains at 6 lines. 
Also the search does not work, if I enter a term that definitely exists, it is not found. 
What can I do? Do you have any tips?

**Comment by Fraggle78** [comment]
"Show 1 to 6 of 6 URLs, count 6 clicks.
In total, tracking 196 links, 6 clicks and more!"

Shows me no matter how many I want to show, anything under 6 lines works correctly.

**Comment by ozh** [comment]
Works for me, I can display 7, 55, or whatever number of links

**Comment by ozh** [comment]
Details on your setup ?

**Comment by Fraggle78** [comment]
Sorry for the late reply.
My setup:
I installed it under docker, I mounted the folder /var/www/html completely and adjusted the script in it, so the url and unique URLs.

I have a test installation, on this one it doesn't work either. When I specify the number of lines no url is displayed.

**Reply by Fraggle78** [reply]
If I want to display 8 lines, it works, with 9 or 10 lines it is only 8 lines. Seems like something is hanging
